package.json

```diff
@@ -27,7 +27,10 @@
     "lint": "eslint --fix --ext .ts .",
     "lint:ci": "eslint -f checkstyle -o reports/checkstyle.xml --ext .ts .",
     "watch": "tsc -w | tee",
-    "clean": "rm -rf node_modules; rm -rf package-lock.json; rm -rf dist"
+    "clean": "rm -rf node_modules; rm -rf package-lock.json; rm -rf dist",
+    "bump:patch": "bump2version patch && npm version --no-git-tag-version patch",
```

Comment:
why both bump2version and npm version? @enys

Reply:
https://linkurious.atlassian.net/browse/LKE-1167 and the attached ticket https://linkurious.atlassian.net/browse/OPS-88

TLDR : limitations in both tools ...